apps/studio/components/interfaces/Auth/Policies/PolicySelection.tsx

```diff
@@ -58,6 +72,33 @@ const PolicySelection = ({
             onClick={onViewEditor}
           />
         </div>
+        <div className="bg-surface-200 rounded-md p-4 flex items-center  gap-2 text-sm">
+          <Info size={16} strokeWidth={2} />
```

Comment:
We use hero icon for Info, Warning and Destructive icons

Reply:
jfyi icons are here - https://github.com/supabase/supabase/blob/feat/highlight-new-rls-assistant/packages/ui/src/components/Admonition/index.tsx#L21

they aren't exported - will bring them out